---
asr_msgs/AsrTypeAndId[] allObjects
string scenePath
